src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/FinancialTransactionCategoryCreateDTO.java

```diff
@@ -1,11 +1,15 @@
 package pl.byczazagroda.trackexpensesappbackend.dto;
 
 import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionType;
+import pl.byczazagroda.trackexpensesappbackend.regex.RegexConstant;
 
-import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Size;
+import javax.validation.constraints.Pattern;
 
-public record FinancialTransactionCategoryCreateDTO(@NotBlank @Size(max = 30, message = "CATEGORY NAME TOO LONG")
-                                                    String name, @NotNull FinancialTransactionType type,
-                                                    @NotNull Long userId) { }
+public record FinancialTransactionCategoryCreateDTO(
+        @NotNull
```

Comment:
Based on the value of 
`RegexConstant.CATEGORY_NAME_PATTERN`
annotation @NotNull is not needed here.

Reply:
There is an empty line, the issue might be with GitHub.